server/parser/src/main/antlr4/org/eclipse/lsp/cobol/core/CobolParser.g4

```diff
@@ -752,8 +752,19 @@ procedureSectionHeader
    : sectionName SECTION integerLiteral?
    ;
 
+sectionOrParagraph
+   : (cobolWord | integerLiteral)
```

Comment:
Idea for the future: These tokens must start in section A, maybe we could create a distinct token that would help make the grammar less ambiguous.